spec/index.html

```diff
@@ -11591,13 +11591,13 @@ <h2>Conformance</h2>
 
     <section id="mediaType">
       <h2>Internet Media Type, File Extension and Macintosh File Type</h2>
-      <p>The Internet Media Type / MIME Type for the SPARQL Query Language is
+      <p>The Internet Media Type (formerly known as MIME Type) for the SPARQL Query Language is
         "<code>application/sparql-query</code>".</p>
       <p>It is recommended that sparql query files have the extension ".rq" (lowercase) on all
         platforms.</p>
       <p>It is recommended that sparql query files stored on Macintosh HFS file systems be given a
         file type of "TEXT".</p>
-      <div class="mime">
+      <div class="mediatype">
```

Comment:
```suggestion
      <div class="mediatype"><div id="mime"></div>
```

Reply:
Apologies - I missed these on the other PRs. I will go back and apply the change on json results and XML results.